### PR TITLE
Add recipe for pywin32-220

### LINKED ISSUE #1935: Add pywin32
Would be good to have a package for `pywin32`.

### PR DESCRIPTION
**Pull request by nicoddemus**
Fix #1935

### CODE REVIEW

**Comment by nicoddemus** [comment]
Heh there's a [spyder](https://github.com/conda-forge/staged-recipes/blob/master/recipes/meta.yaml) recipe in the `recipes` folder which is getting built along.

It should OK to merge this, but we should remove that `meta.yml` from `master`.

**Comment by jakirkham** [comment]
Yeah, we messed up on that one. Sorry about that. Should now be fixed thanks to PR ( https://github.com/conda-forge/staged-recipes/pull/2283 ).

File: recipes/pywin32/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    - win32pipe
+    - win32print
+    - win32process
+    - win32profile
+    - win32ras
+    - win32security
+    - win32service
+    - win32transaction
+    - win32ts
+    - win32wnet
```
Comment (jakirkham) [inline recipes/pywin32/meta.yaml]: This is intense. 😮 Maybe we should move these into a [`run_test.py`]( http://conda.pydata.org/docs/building/recipe.html#conda-recipe-files-overview ) file so as to keep the `meta.yaml` compact.

File: recipes/pywin32/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+about:
+  home: https://sourceforge.net/projects/pywin32
+  license: PSF
```
Comment (jakirkham) [inline recipes/pywin32/meta.yaml]: What version? Would guess PSF 2, but have not checked.

Comment (jakirkham) [inline recipes/pywin32/meta.yaml]: I guess they are not clear about this either. 😕

Also seems to include other libraries that have various licenses. Mainly BSD and LGPL. Also includes SWIG, which is GPL, but I don't think that is a concern as one doesn't link to SWIG. Plus SWIG has an exception for this case.

Comment (jakirkham) [inline recipes/pywin32/meta.yaml]: Was also going to ask about a license file, but can seem to find the appropriate one. So I guess we can pass on that too.

File: recipes/pywin32/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://downloads.sourceforge.net/project/{{ name }}/{{ name }}/Build {{ version }}/{{ name }}-{{ version }}.zip
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup{{ '3' if py3k else '' }}.py install --record=record.txt
```
Comment (jakirkham) [inline recipes/pywin32/meta.yaml]: As another option, which may be a bit cleaner, one can have a multiline script with selectors. Actually now that I think of it `pyinstaller` has a good example of this. 😆

**Comment by jakirkham** [comment]
So I guess only two things to do. Would move the imports to `run_test.py`. Also would change `script` to have two lines (one for each case) with selectors. Otherwise LGTM.

**Comment by nicoddemus** [comment]
Thanks for the review, applied your suggestions and rebased on `master`. 👍

File: recipes/pywin32/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py>=35]
+
+requirements:
+  build:
+    - python  # [win]
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py>=35]
```
Comment (jakirkham) [inline recipes/pywin32/meta.yaml]: Same story as above.

File: recipes/pywin32/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  run:
+    - python
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py>=35]
```
Comment (jakirkham) [inline recipes/pywin32/meta.yaml]: Same story as above.

File: recipes/pywin32/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+build:
+  number: 0
+  script: 
+    - python setup.py install --record=record.txt  # [not py3k]
+    - python setup3.py install --record=record.txt  # [py3k]
+  skip: True  # [not win]
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py>=35]
```
Comment (nicoddemus) [inline recipes/pywin32/meta.yaml]: Oh that is very interesting. My understanding was that you always need those features if your package has C extensions, but what you say makes perfectly sense if Python already declares and tracks the appropriate runtime.

**Comment by nicoddemus** [comment]
As I commented, I thought it was required. I will remove it then.

**Comment by nicoddemus** [comment]
Unless somebody else has comments, this can be merged I guess. 😁

**Comment by nicoddemus** [comment]
Thanks @ocefpaf! (Valeu 👍)

**Comment by ocefpaf** [comment]
> Thanks @ocefpaf! (Valeu 👍) 

Sem probrema :wink: 

PS: Você faz parte do pessoal de Floripa? Em breve to me mudando pra lá. Qq coisa que entra em contato pelo gitter.

**Comment by nicoddemus** [comment]
Beleza, faço parte sim, trabalho na [ESSS](https://github.com/esss).

**Pull request merged**